src/runtime_src/core/tools/xbmgmt2/OO_Config.cpp

```diff
@@ -16,6 +16,7 @@
 // ------ I N C L U D E   F I L E S -------------------------------------------
```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
comment addressed.